packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/package.json.template

```diff
@@ -78,6 +78,7 @@
     "@commitlint/config-conventional": "^17.0.0",
     "@ama-sdk/schematics": "~<%= sdkCoreVersion %>",
     "@ama-sdk/core": "~<%= sdkCoreVersion %>",
+    "@o3r/schematics": "^<%= sdkCoreVersion %>",
```

Comment:
__Open question:__ What if we put it directly in the `@ama-sdk/create`? We already have `@schematics/angular` as dep and other `@angular-devkit` deps. 
Because here it is not really a dev dependency of the sdk.

Reply:
What problem it would fix?
Actually the `@ama-sdk/create` is just the shell script to child process the installation schematics, its dependencies can not benefice to the installation process :S

Reply:
I was thinking that we put a dep in the generated sdk, which is not needed there. And in `@ama-sdk/create` we already have dependencies which are not really needed for the shell script itself , but needed for the generators which are launched.